### DISCUSSION TITLE
Feature to exclude certain folder(s) from being indexed

### DISCUSSION THREAD

**Discussion by add-n2x** [navidrome/navidrome#2923] [Ideas]
In my music library folder I have certain subfolders with music copied from the library. I don't want to re-import them, as they will appear as duplicates. One way would be to create a subfolder for the Navidrome lib, but I do not want the folder hierarchy get too deep.

So could you introduce something like `.navidrome-ignore`, working similarly as `.gitignore`?

P.S. Thank you so much for building such a wonderful piece of software. I spend years finding solution to migrate away from iTunes. Now I have found my destination :-D

**Comment by metalheim** [comment]
That already exists, but I dont think it's documented (as in "mentioned in the official documentation")

https://github.com/navidrome/navidrome/issues/297#issuecomment-630372587

In short: `.ndignore` file will work, but doesnt support the full .gitignore syntax
